tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -355,7 +355,9 @@ protected virtual IAgentWriter GetAgentWriter(TracerSettings settings, IDogStats
 
         private IApi GetApi(TracerSettings settings, IDogStatsd statsd, Action<Dictionary<string, float>> updateSampleRates, IApiRequestFactory apiRequestFactory, bool partialFlushEnabled)
         {
-            if (settings.DataPipelineEnabled)
+            // Due to missing quantization and obfuscation in native side, we can't enable the native trace exporter
```

Comment:
I don't think we should make this change here. We should change the value of `DataPipelineEnabled` based on the value of `StatsComputationEnabled` in `TracerSettings`, so that we know the _actual_ setting value (and why). You can see other examples of this in `TracerSettings`

Reply:
sure,  I moved the logic to TracerSettings as our standard practice.